e2etest/zt_basic_copy_sync_remove_test.go

```diff
@@ -860,6 +862,9 @@ func TestBasic_HashBasedSync_HashDir(t *testing.T) {
 				hashFile := filepath.Join(hashStorageDir, ".asdf.txt"+common.AzCopyHashDataStream)
 				_, err = os.Stat(hashFile)
 				a.AssertNoErr(err)
+
+				isHidden := osScenarioHelper{}.IsFileHidden(a, hashFile)
```

Comment:
Quick check: make sure the data file is also not hidden. Otherwise, lgtm.

Reply:
Yes we shall check both ways, meta shall be hidden and data shall not be hidden (which is the exact issue we are trying to solve here)

Reply:
added